### ISSUE TITLE
Component Accessibility Status: design research

### ISSUE THREAD

**Issue by laurenmrice** [carbon-design-system/carbon-website#3517]
## Acceptance criteria

Determine how the following will be represented on the website.

1. There are no violations when running the automated [Equal Access Checker](https://www.ibm.com/able/toolkit/tools) on default component state.

2. There are no violations when running the automated [Equal Access Checker](https://www.ibm.com/able/toolkit/tools) on more complex component states and variants.

3. There no violations when running automated keyboard navigation tests using Jess/Michaels' tabbing guidelines.

4.  No screenreader violations. (Manual process, leverage Ram's expertise and time.)

#### Design systems
- Explore other design systems for ways they show a11y status information
  - [Adobe spectrum example](https://spectrum.adobe.com/page/checkbox/#Design-checklist)
  - [Supernova checklist example](https://learn.supernova.io/latest/documentation/types-of-blocks/component/component-checklist.html)
  - [Supernova overview table example](https://learn.supernova.io/latest/documentation/types-of-blocks/component/component-overview-table.html)
  - [Primer example](https://primer.style/react/Box)
 

#### PALs
- See if any pals currently show a11y status
  - [Carbon for IBM Products example](https://pages.github.ibm.com/cdai-design/pal/overview/asset-index/)
  - Sustainability?
  - Security?

#### Playback
- Playback research findings to design team

**Comment by sstrubberg** [comment]
Needs to be re-written based on the outcome of https://github.com/carbon-design-system/carbon/issues/13747
